app/uk/gov/hmrc/emailverification/models/PasscodeVerificationRequest.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emailverification.models
+
+import play.api.libs.json.{Json, Reads}
+
+case class PasscodeVerificationRequest(sessionId:String, passcode:String)
```

Comment:
as above, we should get the session ID from the headers

Reply:
yep sure, I just thought we'd asked IV to do something this way, maybe that was a value in the session rather than the id though?